packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -255,60 +251,49 @@ function makeAugmentedSchema(
     const relationshipProperties = interfaces.filter((i) => relationshipPropertyInterfaceNames.has(i.name.value));
     interfaces = interfaces.filter((i) => !relationshipPropertyInterfaceNames.has(i.name.value));
 
-    const relationshipFields = new Map<string, RelationshipField[]>();
+    const relationshipFields = new Map<string, ObjectFields>();
 
     relationshipProperties.forEach((relationship) => {
         const authDirective = (relationship.directives || []).find((x) => x.name.value === "auth");
         if (authDirective) {
             throw new Error("Cannot have @auth directive on relationship properties interface");
         }
 
-        const relationshipFieldMeta = getRelationshipFieldMeta({ relationship, enums });
-
-        if (!pointInTypeDefs) {
-            pointInTypeDefs = relationshipFieldMeta.some((field) => field.typeMeta.name === "Point");
-        }
-        if (!cartesianPointInTypeDefs) {
-            cartesianPointInTypeDefs = relationshipFieldMeta.some((field) => field.typeMeta.name === "CartesianPoint");
-        }
-
-        relationshipFields.set(relationship.name.value, relationshipFieldMeta);
-
-        const fields = {};
-
         relationship.fields?.forEach((field) => {
-            const forbiddenDirectives = ["auth", "relationship"];
+            const forbiddenDirectives = ["auth", "relationship", "cypher"];
```

Comment:
Should there also be a check at the interface level?

```ts
const cypherDirective = (relationship.directives || []).find((x) => x.name.value === "cypher");
if (cypherDirective) {
    throw new Error("Cannot have @cypher directive on relationship properties interface");
}
```

Reply:
The schema validation covers this, but I just did add a test case for it.

Reply:
Of course it does, I forgot that there was separate location definition for interface and object types!